### DISCUSSION TITLE
This a great and you have my full support

### DISCUSSION THREAD

**Discussion by sysadminpower2019** [snarfed/bridgy-fed#888] [General]
While Bluesky is far from perfect and Mastodon is another can of worms. 
I think a project like this that helps to close the gap between different services and protocols can only strengthen the decentralized nature of the web. The only way to take back our data is for us to build our own networks and to share our data freely with one another from any source to another.
This whole opt out / opt in thing is rather ridiculous. If you post anything online anywhere you have to assume responsibility for the post itself. Yes it might suck when its manipulated or feed into some big corporate media but that is exactly the peril of using any social media.
I find that generally twitter style social media leads to anger and frustration and not any real productive talk.
So I am thankful and happy someone has taken the time and effort to try to find a solution.
Of course people should not be forced to collaborate if they don't want to but imagine if e-mail worked like that. Sorry I am opting out of getting emails from your server  you would end up again in a small centralized echo chamber.

**Comment by snarfed** [comment]
Thank you for the kind words and perspective, I appreciate it!
